app/serializers/participant_serializer.rb

```diff
@@ -71,12 +71,12 @@ def validated_trn(user)
     user.teacher_profile.ecf_profile.ecf_participant_eligibility&.eligible_status? || nil
   end
 
-  active_participant_attribute :pupil_premium_uplift do
-    nil # TODO: CPDRP-534 - Share when we know we have the correct information
+  active_participant_attribute :pupil_premium_uplift do |user|
+    user.teacher_profile.ecf_profile&.school&.pupil_premium_uplift?(user.teacher_profile.ecf_profile.cohort.start_year)
```

Comment:
We have that cached on `participant_profiles` too - or so I think. Should we use that instead?

Same with `sparsity_uplift`.

Reply:
@kicferk1 @blackrat Why do T&P cache these?

Reply:
Yes. We should always be using the `participant_profiles` value. Once uplift is set on a participant, it remains set on that participant regardless if they change schools or if the school changes.

Reply:
It was a business rule given to us by product.

Reply:
@blackrat By 'we' do you mean T&P or programme-wide?

Reply:
Discussed offline: previous cached version is obsolete now.

Reply:
> Discussed offline: previous cached version is obsolete now.

No. That's not the case. I was investigating further. It's not effected by the school change, but it should stay with the teacher if they move schools, so caching is still relevant.